staging/src/k8s.io/client-go/tools/watch/retrywatcher_test.go

```diff
@@ -587,11 +587,12 @@ func TestRetryWatcherToFinishWithUnreadEvents(t *testing.T) {
 
 	watcher.Stop()
 
+	maxTime := 20 * time.Millisecond
 	select {
 	case <-watcher.Done():
 		break
-	case <-time.After(10 * time.Millisecond):
```

Comment:
make this a whole second… this isn't a performance test

Reply:
Fixed